x/group/module/module.go

```diff
@@ -155,6 +161,42 @@ func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.Val
 	return []abci.ValidatorUpdate{}
 }
 
+func init() {
+	appmodule.Register(
+		&modulev1.Module{},
+		appmodule.Provide(
+			provideModuleBasic,
+			provideModule,
+		),
+	)
+}
+
+func provideModuleBasic() runtime.AppModuleBasicWrapper {
+	return runtime.WrapAppModuleBasic(AppModuleBasic{})
+}
+
+type groupInputs struct {
+	depinject.In
+
+	KVStoreKey       *store.KVStoreKey
+	Cdc              codec.Codec
+	AccountKeeper    group.AccountKeeper `key:"cosmos.auth.v1.AccountKeeper"`
+	BankKeeper       group.BankKeeper    `key:"cosmos.bank.v1.Keeper"`
+	Registry         cdctypes.InterfaceRegistry
+	MsgServiceRouter *baseapp.MsgServiceRouter
+}
+
+func provideModule(in groupInputs) (keeper.Keeper, runtime.AppModuleWrapper) {
+	groupConfig := group.DefaultConfig()
```

Comment:
These config params need to be in the module config object. We can use defaults if they're empty